src/test/java/com/google/firebase/auth/TenantAwareFirebaseAuthIT.java

```diff
@@ -297,15 +270,15 @@ public void testOidcProviderConfigLifecycle() throws Exception {
             .setEnabled(true)
             .setClientId("ClientId")
             .setIssuer("https://oidc.com/issuer");
-    OidcProviderConfig config = tenantAwareAuth.createOidcProviderConfigAsync(createRequest).get();
+    OidcProviderConfig config = auth.createOidcProviderConfigAsync(createRequest).get();
```

Comment:
Use the tempProviderConfig here and unroll the try-finally?

Reply:
Whoops! Thanks for catching that.